versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = ">= 1.0.0"
```

Comment:
Please copy `versions.tf` here into `examples/complete` or the test will use an older version of terraform instead of using 1.x